acp/source_hub_client.go

```diff
@@ -335,6 +356,71 @@ func (a *sourceHubBridge) CheckDocAccess(
 	}
 }
 
+func (a *sourceHubBridge) AddDocActorRelationship(
+	ctx context.Context,
+	policyID string,
+	resourceName string,
+	docID string,
+	relation string,
+	requestActor identity.Identity,
+	targetActor string,
+) (bool, error) {
+	// Must have all required args.
```

Comment:
suggestion: This comment is not very useful, please either expand it to cover *why* all of the are required (and move it to the func-level), or remove it - the code already states that they are required, without a why this comment is not useful.

Reply:
- [ ] remove redundant comment